### ISSUE TITLE
Dependency jars with trailing zero in minor version number causes classpath mismatch

### ISSUE THREAD

**Issue by norru** [xerial/sbt-pack#91]
Hello,

For compatibility with a legacy system I am building an executable jar with references inside MANIFEST.MF

The following dependency
```
com.esotericsoftware.reflectasm/reflectasm/jars/reflectasm-1.09.jar
```
Gets packed as

```
target/pack/lib/reflectasm-1.9.jar
```

The manifest file correctly adds the original file.

As a result the executable jar fails because it can't reference the classes in the wrongly named ```reflectasm-1.9.jar```

I'm pretty sure I can find one or two workarounds for this, but does look ugly.

EDIT: ```packJarNameConvention := "original"``` seems to produce the correct effect. However, I think the "stealth" version change is a very unintuitive behaviour from the point of view of the general public.

Cheers
Nico

**Comment by xerial** [comment]
0-padding is missing. Is this correct?

**Comment by xerial** [comment]
Current VersionString implementation does not care about the 0-padding. This should be fixed:
https://github.com/xerial/sbt-pack/blob/master/src/main/scala/xerial/sbt/VersionString.scala#L14

**Comment by xerial** [comment]
Fixed in #98
